ani-cli

```diff
@@ -211,10 +217,27 @@ update_history() {
     mv "${histfile}.new" "$histfile"
 }
 
+
 download() {
     case $1 in
-        *m3u8*) ffmpeg -loglevel error -stats -i "$1" -c copy "$download_dir/$2.mp4" ;;
-        *) aria2c --check-certificate=false --continue --summary-interval=0 -x 16 -s 16 "$1" --dir="$download_dir" -o "$2.mp4" --download-result=hide ;;
+        *)
+            if [ "$jellyfin" = "True" ]; then
+                if [ ${#ep_no} -eq 3 ]; then
+                    epstr="E$ep_no"
+                elif [ ${#ep_no} -eq 2 ]; then
+                    epstr="E0$ep_no"
+                elif [ ${#ep_no} -eq 1 ]; then
+                    epstr="E00$ep_no"
+                fi
```

Comment:
This is bad code, but could work for detecting the number of eps I guess.
Zero padding is usually done differently in shell.
Also, this would break for really long anime, like one piece, with >1k episodes